h2o-py/h2o/frame.py

```diff
@@ -3330,3 +3415,45 @@ def _binop(lhs, op, rhs, rtype=None):
     if rtype is not None and res._ex._cache._names is not None:
         res._ex._cache._types = {name: rtype for name in res._ex._cache._names}
     return res
+
+
+
+
+def generatePandaEnumCols(pandaFtrain, cname, nrows):
+    """
+    For a H2O Enum column, we perform one-hot-encoding here and added one more column "missing(NA)" to it.
```

Comment:
Change to:
For an H2O Enum column, we perform one-hot-encoding here and add one more column, "missing(NA)" to it.

Reply:
Done